### DISCUSSION TITLE
Password protection

### DISCUSSION THREAD

**Discussion by Danetag** [vercel/vercel#4085] [General] [answered]
Edit: Answered below https://github.com/vercel/vercel/discussions/4085#discussioncomment-4453713

**Comment by mcsdevv** [comment]
Hey @Danetag, this is something that will shortly be available on Enterprise plans. I'm aware of multiple use cases for this but could you share a little information about what yours would be?

**Reply by wiiiimm** [reply]
Hi @Danetag, we have just enabled the password protection and SSO protection features for Pro and Enterprise plans. Please take a look at https://vercel.com/blog/protecting-deployments for more information.

**Reply by Danetag** [reply]
Thanks @williamli but we actually have a business plan... It's sad to see such a basic feature not available to us (plan costs over $200/month).

Thanks for the heads up!

**Reply by TeddyMathias** [reply]
Agreed! So expensive for something so basic.

**Reply by sorensenjg** [reply]
Yes this is ridiculous at $150/month... most of the other hosting services offer this feature starting in their basic subscription. Vercel offers a great platform, but this is a deal breaker. Please provide this for free.

**Comment by Danetag** [comment]
@msweeneydev Thanks for the quick answer! For us it's a matter of best practices, being able to password protect to make sure that the website is not found accidentally or just shared to the open.

We have an Enterprise plan so it's good news! Thanks

**Reply by mcsdevv** [reply]
Perfect, thanks for sharing the use case!

**Reply by Danetag** [reply]
Actually we have a business plan, would it be part of it?

**Reply by mcsdevv** [reply]
Not currently, but it will be very soon! It'll be available as an additional feature.

**Comment by leerob** [comment, marked as answer]
**Update December 2022**

- [You can now protect preview deployments for free with Vercel authentication](https://vercel.com/blog/security-controls-protected-preview-deployments-passwords)
- [You can also do basic auth with a password fairly easy with Middleware](https://github.com/vercel/examples/tree/main/edge-functions/basic-auth-password)
